gc/base/standard/Scavenger.cpp

```diff
@@ -1011,11 +1010,13 @@ MM_Scavenger::calcGCStats(MM_EnvironmentStandard *env)
 #if defined(OMR_GC_MODRON_CONCURRENT_MARK)
 			if (_extensions->debugConcurrentMark) {
 				OMRPORT_ACCESS_FROM_OMRPORT(env->getPortLibrary());
-            	omrtty_printf("Tenured bytes: %zu\navgTenureBytes: %zu\ntenureBytesDeviation: %f\navgTenureBytesDeviation: %zu\n",
-				tenureAggregateBytes,
-				scavengerGCStats->_avgTenureBytes,
-				tenureBytesDeviation,
-				scavengerGCStats->_avgTenureBytesDeviation);
+				omrtty_printf("Tenured bytes: %zu\navgTenureBytes: %zu\ntenureBytesDeviation: %f\navgTenureBytesDeviation: %zu\ninitialFree: %zu\navgInitialFree: %zu\n",
```

Comment:
first parameter should start on a new line

Reply:
https://github.com/eclipse-omr/omr/blob/master/doc/CodingStandard.md#function-parameter-list-does-not-fit-on-one-line

Reply:
```
omrtty_printf(
		"Tenured bytes: %zu\navgTenureBytes: %zu\ntenureBytesDeviation: %f\navgTenureBytesDeviation: %zu\ninitialFree: %zu\navgInitialFree: %zu\n",